numpy/_core/shape_base.py

```diff
@@ -233,9 +233,11 @@ def vstack(tup, *, dtype=None, casting="same_kind"):
 
     Parameters
     ----------
-    tup : sequence of ndarrays
+    tup : ndarray or sequence of ndarrays
         The arrays must have the same shape along all but the first axis.
-        1-D arrays must have the same length.
+        1-D arrays must have the same length. In the case of a single
+        array_like input, it will be treated a` a sequence of `shape[0]`
+        arrays, each of shape `shape[1:]`.
```

Comment:
`shape` is not defined. To avoid having to define `shape`, what about expressing this in English rather than code?

```suggestion
        array_like input, it will be treated as a sequence of arrays; i.e.,
        each element along the zeroth axis is treated as a separate array.
```